src/components/thumbnail/thumbnail-document-item.tsx

```diff
@@ -27,9 +27,10 @@ export const ThumbnailDocumentItem = observer((props: IProps) => {
   const handleDocumentStarClick = (e: React.MouseEvent<HTMLDivElement>) => {
     onDocumentStarClick && onDocumentStarClick(document);
   };
+  const isSoftDeleted = document.getProperty("softDelete") === "true";
   return (
     <div
-      className="list-item"
+      className={`${isSoftDeleted ? "soft-delete-remove-thumbnail" : "list-item" }`}
```

Comment:
I think not rendering the item will be preferable to rendering it with `display: none`, so your other branch is closer to the mark. That said, I'd be inclined to handle it in `app-config.json` rather than explicitly changing the code in the `<DocumentsSection/>`. The advantage is that then all properties are handled consistently rather than some properties having special implementation details. The advantage of your approach of putting it in code is that it doesn't have to be specified multiple times in `app-config.json`. That said, I could imagine a feature request down the road to show deleted documents in their own section where it's possible to "undelete" them, which the property approach could support. So I still find myself leaning toward the `app-config.json` approach.